### PR TITLE
SEO-172143-Windowsforms-links-changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK ,
Please review the changes made.
Regards,
Scholar.

### CODE REVIEW

File: WindowsForms/Grouping/Getting-Started.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This section illustrates the step-by-step procedure to create the following plat
 
 1. Open Microsoft Visual Studio. Go to File menu and click New Project. In the New Project dialog, select Windows Forms Application template, name the project and click OK. 
 
-   ![WindowsForms Grouping Creating a Windows Application](Getting-Started_images/Getting-Started_img1.jpeg)
+   ![WindowsForms Grouping Creating a Windows Application](getting-started_images/windowsforms-create-windows-application.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/Grouping/Getting-Started.md:24]: Control name is missing in image name.
image
 check and update the control name in all places.

File: WindowsForms/Html-Viewer/Getting-Started.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ Me.Controls.Add(Me.htmluiControl1)
 
 {% endtabs %}
 
-![HTMLUIControl added using code](Getting-Started_images/GettingStarted-img2.png)
+![HTMLUIControl added using code](getting-started_images/windowsforms-adding-code.png)
```
Comment (MallikaRK) [inline WindowsForms/Html-Viewer/Getting-Started.md:112]: Image name don't have the control name. Check and update the control name in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Html-Viewer/Getting-Started.md [hunk 3]
```diff
@@ -39,15 +39,15 @@ The HTMLUIControl can be added to the application by dragging it from the toolbo
    * Syncfusion.Scripting.Base.dll
    * Syncfusion.Shared.Base
 
-![Search html ui control in toolbox](Getting-Started_images/GettingStarted-img1.png)
+![Search HTML UI control in toolbox in WindowsForms](getting-started_images/windowsforms-html-viewer-search-html-ui-control-toolbox.png)
 
-![Drag an drop the html ui control in application](Getting-Started_images/GettingStarted-img5.png)
+![Drag an drop the HTML UI  control in WindowsForms application](getting-started_images/windowsforms-html-viewer-drag-drop-html.png)
```
Comment (MallikaRK) [inline WindowsForms/Html-Viewer/Getting-Started.md:44]: Image name has repeated html keyword.
Change like as below,
adding-html-viewer-control-through-designer

File: WindowsForms/groupview/Overview.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ documentation: ug
 
 The [GroupView](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Tools.GroupView.html) control implements a selectable list that can display a collection of items, where each item is represented by an image and a descriptor. It is capable of displaying items with large or small icons in various combinable styles such as the default selectable style, button-type selection, full-item select and an icon-only flow view mode. 
 
-![Overview of GroupView](Overview_images/overview.png)
+![Overview of Group View in WindowsForms](overview_images/windowsforms-group-viewoverview.png)
```
Comment (MallikaRK) [inline WindowsForms/groupview/Overview.md:13]: recheck the image name. give hyphen properly.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Html-Viewer/Getting-Started.md (hunk 3 above)
Comment (Scholar-jaruha) [reply WindowsForms/Html-Viewer/Getting-Started.md:44]: changed

File: WindowsForms/groupview/Overview.md (hunk 4 above)
Comment (Scholar-jaruha) [reply WindowsForms/groupview/Overview.md:13]: Added hyphen in the image name

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**